src/controllers/socialController.js

```diff
@@ -0,0 +1,102 @@
+import jwt from 'jsonwebtoken';
+import {
+  config
+} from 'dotenv';
+import db from '../models';
```

Comment:
This is not how we have been importing models in the codebase. Please, edit this.